di_website/templates/includes/forms/publication-filter.html

```diff
@@ -1,5 +1,25 @@
 <form method="get" action="{{self.pageurl}}" class="form">
```

Comment:
You know you could have just added the `resources-filters` class on this form element instead of on every form field ... in the css, it'd be `.resources-filters .form-field--inline-five`

Reply:
This makes things neater, updated and pushed